tests/app/main/test_frameworks.py

```diff
@@ -2675,16 +2732,8 @@ def _setup_data_api_client(self, data_api_client, framework_status, framework_sl
         "framework_slug,declaration,decl_valid,prefill_fw_slug,expected_sections",
         _g7_parametrization
     )
-    def test_display_open(
-        self,
-        data_api_client,
-        framework_slug,
-        declaration,
-        decl_valid,
-        prefill_fw_slug,
-        expected_sections,
-    ):
```

Comment:
Another nice catch, do you have a code cleanup plugin for this or just caught it by site?